.gitignore

```diff
@@ -0,0 +1 @@
+_site
```

Comment:
This directory is generated locally whenever you serve with 11ty.  We don't need to commit this to the repo for now I think.